HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponent/PaymentInfoViewController.swift

```diff
@@ -0,0 +1,349 @@
+        label.font = viewModel.payBillsTitleFont
+        label.textColor = viewModel.payBillsTitleTextColor
+        label.lineBreakMode = .byWordWrapping
+        label.numberOfLines = 0
+        label.textAlignment = .left
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.lineHeightMultiple = Constants.payBillsTitleLineHeight
+        label.attributedText = NSMutableAttributedString(string: viewModel.payBillsTitleText,
+                                                         attributes: [NSAttributedString.Key.paragraphStyle: paragraphStyle])
+        return label
+    }()
+    
+    private lazy var payBillsDescriptionTextView: UITextView = {
+        let textView = UITextView()
+        textView.translatesAutoresizingMaskIntoConstraints = false
+        textView.isScrollEnabled = false
+        textView.isEditable = false
+        textView.textContainerInset = .zero
+        textView.textContainer.lineFragmentPadding = 0
+        textView.isUserInteractionEnabled = true
+        textView.backgroundColor = .clear
+        textView.attributedText = viewModel.payBillsDescriptionAttributedText
+        return textView
+    }()
+    
+    private lazy var questionsTitleLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = viewModel.questionsTitleFont
+        label.textColor = viewModel.questionsTitleTextColor
+        label.lineBreakMode = .byWordWrapping
+        label.numberOfLines = 0
+        label.textAlignment = .left
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.lineHeightMultiple = Constants.questionsTitleLineHeight
+        label.attributedText = NSMutableAttributedString(string: viewModel.questionsTitleText,
+                                                         attributes: [NSAttributedString.Key.paragraphStyle: paragraphStyle])
+        return label
+    }()
+    
+    private lazy var questionsTableView: UITableView = {
+        let tableView = UITableView()
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.register(PaymentInfoAnswerTableViewCell.self,
+                           forCellReuseIdentifier: PaymentInfoAnswerTableViewCell.identifier)
+        tableView.separatorStyle = .singleLine
+        tableView.backgroundColor = .clear
+        tableView.showsVerticalScrollIndicator = false
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        tableView.isScrollEnabled = false
+        tableView.rowHeight = UITableView.automaticDimension
+        tableView.estimatedRowHeight = Constants.questionTitleHeight
+        tableView.estimatedSectionHeaderHeight = Constants.questionTitleHeight
+        tableView.estimatedSectionFooterHeight = 1.0
+        if #available(iOS 15.0, *) {
+            tableView.sectionHeaderTopPadding = 0
+        } else {
+            // Fallback on earlier versions
```

Comment:
Since iOS 15, UITableView contains a new property called [sectionHeaderTopPadding](https://developer.apple.com/documentation/uikit/uitableview/3750914-sectionheadertoppadding) which specifies the amount of padding above each section header.

Yes, we need this. Otherwise we will have extra padding above each section from UITableView.